migrations/deleteDuplicateRecordsAcrossCollections/findDuplicateRecords.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * name : findDuplicateRecords.js
+ * author : Saish Borkar
+ * created-date : 1 Jul 2024
+ * Description : Finds duplicate records from project collection having same solutionId and userId
+ */
+
+const path = require("path");
+let rootPath = path.join(__dirname, "../../");
+require("dotenv").config({ path: rootPath + "/.env" });
+const {validate : uuidValidate,v4 : uuidV4} = require('uuid');
+let _ = require("lodash");
+let mongoUrl = process.env.MONGODB_URL;
+let dbName = mongoUrl.split("/").pop();
+let url = mongoUrl.split(dbName)[0];
+var MongoClient = require("mongodb").MongoClient;
+
+var fs = require("fs");
+
+(async () => {
+  let connection = await MongoClient.connect(url, { useNewUrlParser: true });
+  
```

Comment:
we dont need this @borkarsaish65

Reply:
I tried removing this , it gives an error. All other scripts also have the let connection declaration.